packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -81,15 +82,16 @@
         
         "referenceRange":
         [
+        {{!-- values here are type SimpleQuantity: https://www.hl7.org/fhir/R4/datatypes.html#SimpleQuantity --}}
           {
             "low":
                 {
-                    "value":"{{observationEntry.referenceRange.observationRange.value.low.value}}",
+                    "value":"{{removeNonNumeric (decodeHtmlEntities observationEntry.referenceRange.observationRange.value.high.value)}}",
```

Comment:
oversight? `.observationRange.value.high.value` should probably reference `value.low.value`